resources/ui/pointing-page.blp

```diff
@@ -77,5 +77,14 @@ Box content_box {
         hexpand: true;
       }
     }
+
+    Adw.ComboRow t_send_events_row {
+      title: _("Send events touchpad");
+      model: StringList { strings [
+        _("Enabled"),
+        _("Disabled"),
+        _("Disabled-on-external-mouse"),
+      ]};
+    }
```

Comment:
Thank you very much for the work!

But this is not ready for merge yet.

![Screenshot from 2024-07-08 10-24-51](https://github.com/gdm-settings/gdm-settings/assets/67191572/be262f3b-497e-4916-9178-880e9cfb6a68)
![Screenshot from 2024-07-08 10-25-04](https://github.com/gdm-settings/gdm-settings/assets/67191572/fcb178ff-770f-4178-abf6-6350dbe1a6d2)

This is not a user-friendly way to present the setting. We need to change how we present the setting, specially the wording needs improvement.

This should either be
1. A toggle/switch
   **Title**: Disable On External Mouse
   **Subtitle**: Disable the touchpad when an external mouse is connected, enable it otherwise.
   This toggle would switch between `enabled` and `disabled-on-external-mouse` values.

Or

2. The very first setting in touchpad section, or maybe even as a suffix to the touchpad `PreferencesGroup`
   * It should have all three options, but the user-presentable string for `disabled-on-external-mouse` should be more concise yet still easy to understand for users.
   * If it is its own setting, the title (and optionally subtitle) should be easy to understand.
   * It is not a deal-breaker, but when the user chooses `disabled` for this setting, all other touchpad settings should become `inactive` (dimmed, and not changeable by the user).

I would much prefer the option 2 here, as it gives the user the most choice. However, the details of that one are not finished yet. So, we'll have to discuss those if we go with that option. And option 1 is also fine for now.